README.md

```diff
@@ -38,7 +38,10 @@ To configure your Ubiquibot to run this plugin, add the following to the `.ubiqu
   with:
     reviewDelayTolerance: "3 Days"
     taskStaleTimeoutDuration: "30 Days"
-    maxConcurrentTasks: 3
+    maxConcurrentTasks: # Default concurrent task limits per role.
+      admin: 10
```

Comment:
I feel like admins shouldn't have limits but to be honest I feel this feature has no teeth because we can just use the GitHub UI. 

It would be more interesting if it unassigns when we self assign and we hit the limit.

Reply:
@0x4007 we can change UI assignments to follow the same logic as `/start` so user would get unassigned if they are doing too many tasks, even when forcing it through the UI.